modules/nw-sriov-dpdk-example-mellanox.adoc

```diff
@@ -80,6 +80,8 @@ spec:
 =====
 See the `Configuring SR-IOV additional network` section for detailed explanation on each option in `SriovNetwork`.
 =====
++
+An optional library, app-netutil, provides several API methods for gathering network information about a pod from within a container running within that pod.
```

Comment:
Please add the following link for reference: https://docs.openshift.com/container-platform/4.8/networking/hardware_networks/about-sriov.html#nw-sriov-app-netutil_about-sriov